SPECS/cloud-init/cloud-init.spec

```diff
@@ -1,8 +1,8 @@
 %define python3_sitelib /usr/lib/python3.7/site-packages
 
 Name:           cloud-init
-Version:        19.1
-Release:        6%{?dist}
+Version:        21.2
+Release:        0%{?dist}
```

Comment:
we've been starting all releases at 1; could you do the same?

Reply:
Fixed.